CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * MapboxNavigation now requires [MapboxMaps v10._x_](https://github.com/mapbox/mapbox-maps-ios/releases/tag/v10.2.0). ([#3665](https://github.com/mapbox/mapbox-navigation-ios/pull/3665))
 * MapboxCoreNavigation now requires [MapboxDirections v2.2.0-rc.1](https://github.com/mapbox/mapbox-directions-swift/releases/tag/v2.2.0-rc.1). ([#3683](https://github.com/mapbox/mapbox-navigation-ios/pull/3683))
 * MapboxCoreNavigation now requires [MapboxNavigationNative v83._x_](https://github.com/mapbox/mapbox-navigation-native-ios/releases/tag/83.0.0). ([#3683](https://github.com/mapbox/mapbox-navigation-ios/pull/3683))
+* MapboxCoreNavigation now depends on [MapboxDirections v2.2.0](https://github.com/mapbox/mapbox-directions-swift/releases/tag/v2.2.0). ([#3694](https://github.com/mapbox/mapbox-navigation-ios/pull/3694))
```

Comment:
There is already an entry on line 8, you can edit it instead of adding this one.

Reply:
Oh yes. I'll correct this one.